src/uct/sm/mm/base/mm_md.c

```diff
@@ -67,22 +67,23 @@ ucs_status_t uct_mm_seg_new(void *address, size_t length, uct_mm_seg_t **seg_p)
     return UCS_OK;
 }
 
-void uct_mm_md_query(uct_md_h md, uct_md_attr_t *md_attr, uint64_t max_alloc)
+void uct_mm_md_query(uct_md_h md, uct_md_attr_v2_t *md_attr, uint64_t max_alloc)
 {
     memset(md_attr, 0, sizeof(*md_attr));
 
-    md_attr->cap.flags            = UCT_MD_FLAG_RKEY_PTR |
-                                    UCT_MD_FLAG_NEED_RKEY;
-    md_attr->cap.max_reg          = 0;
-    md_attr->cap.max_alloc        = 0;
-    md_attr->cap.alloc_mem_types  = UCS_BIT(UCS_MEMORY_TYPE_HOST);
-    md_attr->cap.cache_mem_types  = UCS_BIT(UCS_MEMORY_TYPE_HOST);
-    md_attr->cap.access_mem_types = UCS_BIT(UCS_MEMORY_TYPE_HOST);
-    md_attr->cap.detect_mem_types = 0;
+    md_attr->flags            = UCT_MD_FLAG_RKEY_PTR |
```

Comment:
can we remove the memset at line 72?

Reply:
@yosefe this wasn't introduced in this PR. Should I remove still remove the memset?

Reply:
Right, but this PR is resetting md_attr to 0 before calling TL operation. So now it can be removed.